static/src/javascripts/projects/common/modules/navigation/search.js

```diff
@@ -47,6 +47,21 @@ define([
                 mediator.emit('modules:search');
             });
 
+            bean.on(document, 'keydown', '.gsc-input', function (e) {
```

Comment:
We've tried to optimise this, but unfortunately, it looks like this block gets reconstructed every time.